org.goldcoin.goldcoin-qt.metainfo.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.goldcoin.goldcoin-qt</id>
```

Comment:
this doesn't seem to be the same as the project domain website. So to me it doesn't fit the requirements.

Reply:
Is this suitable for flatpak requirements: **org.goldcoinproject.goldcoin-qt**